### PR TITLE
Add two sentences on openness in research

### PR DESCRIPTION
**Pull request by slochower**
Refs https://github.com/greenelab/meta-review/issues/66

Rework the first sentence and add two more sentences on the specific benefits of open research, with two more citations.

The text is rough, but this is a start.

### CODE REVIEW

File: content/02.main-text.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ## Introduction {.page_break_before}
 
-Openness in research --- which includes sharing code, data, and manuscripts --- benefits the researchers who practice it [@doi:10.7554/eLife.16800], their scientific peers, and the public.
+Openness in research benefits the researchers who practice it [@doi:10.7554/eLife.16800], their scientific peers, and the public.
```
Comment (agitter) [inline content/02.main-text.md:3]: We could add the new article https://www.nature.com/articles/s41467-018-05227-z here: `their scientific peers [@doi:10.1038/s41467-018-05227-z]`

I'm also looking for a reference that discusses why open research benefits the public.  There is a brief statement at https://www.plos.org/open-access/
> Much scientific and medical research is paid for with public funds. Open Access allows taxpayers to see the results of their investment.

but there may be something with more depth.

**Review by agitter — COMMENTED** [review]
This looks good to me, but I'll wait for @dhimmel before deciding to merge.  See if you like the optional new reference.

File: content/02.main-text.md (hunk 1 above)
Comment (slochower) [reply content/02.main-text.md:3]: I saw the point about funding as well. I just wasn't sure how universally it applies to other countries.

File: content/02.main-text.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
+Improving the availability of code and data helps ensure the methods are transparent and the results can be reproduced  [@doi:10.1038/s41562-016-0021].
+Publishing in an open access journal increases the number of readers and reviewers [@doi:10.1371/journal.pone.0000308].
```
Comment (dhimmel) [inline content/02.main-text.md:5]: This citation (https://doi.org/10.1371/journal.pone.0000308) is about publicly sharing data increasing citations, not about the open access citation advantage.

File: content/02.main-text.md (hunk 1 above)
Comment (dhimmel) [reply content/02.main-text.md:3]: > We could add the new article https://www.nature.com/articles/s41467-018-05227-z here

@agitter the article behind that link is [Assessment of the impact of shared brain imaging data on the scientific literature](https://doi.org/10.1038/s41467-018-04976-1). Personally, I find this study severely shortsighted in that it judges the value of a study by the impact of the journal it was published in:

> Several articles that used INDI data have been published in high-impact specialized journals (e.g., Biological Psychiatry and Neuron) and general-interest journals (e.g., Proceedings of the National Academy of Sciences and Nature Communications)

This sort of reasoning is NOT OPEN and runs counter to the longterm vision of Manubot. In the longterm, we want Manubot to enable scholarly publishing & communication without journals. This study just shows that studies that reuse brain imaging data can be published in high impact journals.

**Review by dhimmel — COMMENTED** [review]
I'd like to flesh out the discussion a bit more and then come back to these two sentences, and see where in the manuscript they fit best, if at all.

My worry is that they currently weaken the introduction, which succinctly argues the benefit of open manuscript preparation for review articles. If we want to discuss open science in more depth at the beginning, we should make sure it leads the reader to the observation that open manuscript drafting is the next frontier of open science. And that open drafting will complement the other progress being made towards open science (open access, open data, open source, article-level metrics, etcetera).

**Comment by slochower** [comment]
> My worry is that they currently weaken the introduction, which succinctly argues the benefit of open manuscript preparation for review articles. If we want to discuss open science in more depth at the beginning, we should make sure it leads the reader to the observation that open manuscript drafting is the next frontier of open science.

Do you think the introduction would benefit from additional sentences on open science / open research, as mentioned in #66?

File: content/02.main-text.md (hunk 1 above)
Comment (agitter) [reply content/02.main-text.md:3]: I also did not like the focus on high-impact journals in the original article.  I thought the editorial was more broad and slightly better, but let's skip it.

File: content/02.main-text.md (hunk 2 above)
Comment (slochower) [reply content/02.main-text.md:5]: There are better ones. This seems to be one of the most cited articles on the open access citation advantage -- http://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.0040157 -- but the articles it considered are quite old, from 2004-2005. I only skimmed this one, but it is more recent and seems to make the same points (see section 2.1 and Figure 1): https://cdn.elifesciences.org/articles/16800/elife-16800-v1.pdf

**Comment by agitter** [comment]
> Do you think the introduction would benefit from additional sentences on open science / open research, as mentioned in #66?

@dhimmel has some good ideas for this, so we'd like to merge what you have here and let him add to it.

**Pull request merged**